src/Services/Air/AirFormat.js

```diff
@@ -45,6 +56,8 @@ function formatSegment(segment) {
     airline: segment.Carrier,
     flightNumber: segment.FlightNumber,
     uapi_segment_ref: segment.Key,
+    codeshare,
```

Comment:
I would name it `isCodeshare` to indicate that it's a boolean flag

Reply:
changed to isCodeshare